sdcm/scylla_bench_thread.py

```diff
@@ -111,7 +111,13 @@ def __init__(self, stress_cmd, loader_set, timeout, node_list=None, round_robin=
                          stop_test_on_failure=stop_test_on_failure)
         if credentials and 'username=' not in self.stress_cmd:
             self.stress_cmd += " -username {} -password {}".format(*credentials)
-        self.stress_cmd += ' -error-at-row-limit 1000'  # make it fail after having 1000 errors at row
+
+        if not ('-error-at-row-limit' in self.stress_cmd
+                or '-error-limit' in self.stress_cmd
+                or '-retry-number' in self.stress_cmd and int(
+                    self.stress_cmd.split("-retry-number")[-1].replace("=", "").strip().split(" ")[0]) > 1):
```

Comment:
won't a regex would work a bit better for this one ?

Reply:
What exactly will be better? Both will work, no side effects...

Reply:
readability?

Reply:
Done